src/Console/CastorFileNotFoundApplication.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Castor\Console;
+
+use Castor\Console\Command\InitCommand;
+use Symfony\Component\Console\Application as SymfonyApplication;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/** @internal */
+class CastorFileNotFoundApplication extends SymfonyApplication
+{
+    public function __construct(
+        private readonly \RuntimeException $e,
+    ) {
+        parent::__construct(Application::NAME, Application::VERSION);
+    }
+
+    public function run(InputInterface $input = null, OutputInterface $output = null): int
+    {
+        $this->add(new InitCommand($this->e));
+        $this->setDefaultCommand('init');
```

Comment:
We can not set the second argument singleCommand to true because it breaks the native commands from sf